telemetry/definitions/commonDefinitions.json

```
@@ -60,6 +55,16 @@
             "allowedValues": ["editor", "viewer", "filesystem"],
             "description": "The IDE or OS component used for the action. (Examples: S3 download to filesystem, S3 upload from editor, ...)"
         },
+        {
+            "name": "isTextFilter",
+            "type": "boolean",
+            "description": "A text based filter was used"
+        },
+        {
+            "name": "isTimeFilter",
```

```suggestion
            "name": "hasTimeFilter",
```

Would be interesting to see what the common ranges are for filtering. Just don't know what the best way is to encode this.

---

Yeah I agree that'll be intereseting. Maybe having some default intervals enums as optional metadata field like [15 min, 30 min, 1 hr, 12 hr, 1day, more than a day]